frontend/src/app/AppContext.ts

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { StorageClassKind } from '~/k8sTypes';
-import { DashboardConfigKind } from '~/k8sTypes';
+import { StorageClassKind, DashboardConfigKind } from '~/k8sTypes';
```

Comment:
@alexcreasy very minor nit: maybe do the listing alphabetically, i.e.

`import { DashboardConfigKind, StorageClassKind } from '~/k8sTypes';`